feat(hw7)

karaniecV opened the pull request:
промисы не поддаются))

karaniecV committed:
feat(hw7)

промисы не поддаются))

doroshkovd commented on hw7/hw7.js:
```diff
@@ -0,0 +1,82 @@
+      бачнасць ${Math.round(result['daily']['data']['1']['visibility'])}км,
+      цiск ${result['daily']['data']['1']['pressure']} мм рт. сл.
+      ${result['daily']['data']['1']['summary']}`))
+
```
этот шаблон лучше было вынести в функцию какую нибудь например renderRespone

doroshkovd commented on hw7/hw7.js:
```diff
@@ -0,0 +1,82 @@
+
+function Timer(limit) {
+    let intervalId = null;
+
```
тут надо добавить let value = limit

doroshkovd commented on hw7/hw7.js:
```diff
@@ -0,0 +1,82 @@
+
+    this.start = function () {
+        return new Promise((resolve, reject) => {
+            let value = limit.value;
```
тут неправильно использовать value, вдруг я в таймер закину просто число, и это надо перенести из метода

doroshkovd reviewed (changes requested):
https://jsfiddle.net/doroshkovd/u0fgmae7/6/ - вот пример с паузой 
https://jsfiddle.net/doroshkovd/u0fgmae7/12/ - это пример запуска один за другим
я пока не апрувлю попробуй еще раз разобраться если не получиться подойди на занятии вместе глянем

karaniecV committed:
fix(hw7)

Тask 1: добавлена функция renderResponce.
Task 2: исправлен метод Promice;
вынести let value = limit.value за пределы метода Promice не ударось, т.к. при старте кода "value" инициализируется как пустая строка, после чуго данные из "input" не получаетс передать в функцию "this.start"
